mscore/musescore.cpp

```diff
@@ -6314,6 +6314,14 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   _styleDlg->setScore(cs);
             _styleDlg->exec();
             }
+	  else if (cmd == "edit-header-footer") {
+			if (!_styleDlg)
+				  _styleDlg = new EditStyle{ cs,this };
```

Comment:
Shouldn't this be `_styleDlg = new EditStyle(cs, this);`?

Reply:
It is done the same way a few lines up. (except for an extra space added there)

Reply:
Someone from the internet: Prefer {} initialization over alternatives unless you have a strong reason not to. https://stackoverflow.com/questions/18222926/why-is-list-initialization-using-curly-braces-better-than-the-alternatives

Reply:
@Howard-C Indeed, using an initializer list is advised by Stroustrup to improve performance and protect against bugs. There is an argument to be made for readability (because a lot of very common programming languages don't have this syntax) but it's used throughout the MuseScore codebase, so I don't see a problem with that.